### ISSUE TITLE
The weight details for the census loss and Charbonnier L1 loss

### ISSUE THREAD

**Issue by yingc1309** [JunHeum/ABME#19]
I am working on re-implementing the training processing on our own dataset for comparison experiments.
The performance of retrained model appears unusual when using the sum of census loss and Charbonnier L1 loss, as mentioned in the paper.
![image](https://github.com/JunHeum/ABME/assets/156304697/734338fc-3467-448e-ab8a-4a1a7285f811)
Do we also need to use the optical flow smoothness constraint loss, similar to BMBC?
It would be great if you could provide the details of loss function and the weight details.
